Tests/FluentAssertions.Specs/Types/MethodInfoSelectorSpecs.cs

```diff
@@ -367,6 +396,16 @@ internal class TestClassForMethodSelectorWithNonInheritableAttributeDerived : Te
         public override void PublicVirtualVoidMethodWithAttribute() { }
     }
 
+    internal class TestClassForMethodSelectorWithAsyncAndNonAsyncMethod
+    {
+        public async virtual void PublicVirtualVoidAsyncMethod()
+        {
+            await Task.Yield();
+        }
+
+        public virtual void PublicVirtualVoidNotAsyncMethod() { }
```

Comment:
```suggestion
        public async Task PublicAsyncMethod()
        {
            await Task.Yield();
        }

        public void PublicNotAsyncMethod() { }
```

Reply:
I didn't see the "Commit suggestion" button, nice feature.